crates/beerus-rpc/src/lib.rs

```diff
@@ -181,7 +184,7 @@ impl BeerusRpcServer for BeerusRpc {
             .get_gas_price()
             .await
             .map_err(|_| Error::from(BeerusApiError::ContractError))?;
-        Ok(gas_price.to_string())
+        Ok(format!("0x{gas_price:x}"))
```

Comment:
We need to check if all methods that should return hex considering the Ethereum rpc specs: 

Indeed return hex, and does that should not, do not, so we are 100% compliant.